### DISCUSSION TITLE
GLTF download from MeshExporter sample not working

### DISCUSSION THREAD

**Discussion by VescanAlexandru** [iTwin/itwinjs-core#4622] [Q&A]
Hi all,

I am trying to download the GLTF file from Mesh Export in my Sandbox.
Export to GLTF button works, it starts and finishes the export but when I click the Download Result I get this error:

AuthenticationFailedServer failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature. RequestId:3e5ae7bf-901e-0051-1b29-f4357b000000 Time:2022-11-09T10:52:49.4807785ZSignature did not match. String to sign used was rl 2022-11-09T10:47:43Z 2022-11-09T11:22:43Z /blob/gltfprodeussa01/$root https,http 2020-10-02 c

I have not modified any code and I have simply used the MeshExport sample provided. I worked last week

Any thoughts on that?

**Comment by calebmshafer** [comment]
Hey @VescanAlexandru, we will look into the issue. We have made a few updates to the Mesh Export service in the past couple weeks so there is a change most likely needed in the sample.

cc: @diegopinate

**Comment by diegopinate** [comment]
@VescanAlexandru, we have updated the sample. Let us know if you find any more issues.
